storage/filestore/dir.go

```diff
@@ -77,10 +79,19 @@ func (dir *Dir) DeleteTemporary(file *os.File) error {
 	return errs.Combine(closeErr, os.Remove(file.Name()))
 }
 
-// refToPath converts blob reference to a filepath
-func (dir *Dir) refToPath(ref storage.BlobRef) string {
-	hex := hex.EncodeToString(ref[:])
-	return filepath.Join(dir.blobdir(), hex[0:2], hex[2:])
+// blobToPath converts blob reference to a filepath
+func (dir *Dir) blobToPath(ref storage.BlobRef) string {
+	namespace := pathEncoding.EncodeToString(ref.Namespace)
+	key := pathEncoding.EncodeToString(ref.Key)
+	return filepath.Join(dir.blobdir(), namespace, key[:2], key[2:])
+}
+
+// blobToTrashPath converts blob reference to a filepath
```

Comment:
better comment to understand the difference between `blobToTrashPath ` and `blobToPath `